docs/k8s-helm.md

```diff
@@ -213,6 +217,10 @@ Due to the [limitation of Kubernetes Statefulset](https://github.com/kubernetes/
 
 ### Upgrade Timeplus Enterprise
 
+#### Do not attempt to upgrade across multiple major versions at a time
```

Comment:
the major version in helm charts are different than TPE major versions, maybe you can explain a bit or give an example

Reply:
or mention the helm chart version follows the semver

Reply:
updated